code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/dao/impl/AuthDaoImpl.java

```diff
@@ -200,7 +200,7 @@ public Account createAccount(String name, String kind, String externalId, String
     }
 
     @Override
-    public Identity getIdentity(Long id) {
+    public Identity getIdentity(Long id, boolean obfuscate) {
```

Comment:
This method was modified with this flag so that when called we know if the returned value will be the one for server use or client.